content/r/concepts/data-frames/terms/str/str.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'str()'
+Description: 'Displays the internal structure of a data frame.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-r'
+  - 'paths/computer-science'
+---
+
+The **`str()`** function displays the internal structure of a data frame.
+
+## Syntax
+
+```pseudo
+str(df)
+```
+
+- `df`: The data frame to call the function on.
```

Comment:
change df to dataframe to stay consistent, and change bullet point into sentence